components/dashboard/src/configurations/detail/ConfigurationName.tsx

```diff
@@ -50,23 +51,35 @@ export const RepositoryNameForm: FC<Props> = ({ configuration }) => {
     );
 
     return (
-        <form onSubmit={updateName}>
+        <form onSubmit={updateName} className="border border-gray-300 dark:border-gray-700 rounded-xl p-4">
```

Comment:
Would be great to componentize this in some of the followup pages if we find it's basically the same.